### PR TITLE
Mentor system return

### PR DESCRIPTION
**Pull request by Poojawa**
:cl: Poojawa
add: Re-added Mentor systems
tweak: Mentors have been moved to bare minimum admin flags, if just to simplify some aspects.
tweak: Mentors have a pared down version of the ticket system, seperate from ahelp tickets. All actions are observable by administration. Mentors found to be metagaming with this will be demoted on the spot, further punishment depends on rule break guidelines.
/:cl:

Hooray.

Holy fuck this is a pain in the ass to do. If the ticket system works as planned, I'll probably offer it to Hippie, as the source of this code again is from Hippie Station's

### CODE REVIEW

**Comment by Poojawa** [comment]
wew. need to actually finish implementing the ticket system.

**Comment by jughu** [comment]
Mentor me mom

File: code/__DEFINES/admin.dm [hunk 1]
```diff
@@ -6,7 +6,8 @@
 #define MUTE_PRAY		4
 #define MUTE_ADMINHELP	8
 #define MUTE_DEADCHAT	16
-#define MUTE_ALL		31
+#define MUTE_ALL		32
```
Comment (silicons) [inline code/__DEFINES/admin.dm:9]: shouldn't this be all of the above combined?

File: code/__DEFINES/citadel_defines.dm [hunk 2]
```diff
@@ -68,6 +68,13 @@
 //Individual logging define
 #define INDIVIDUAL_LOOC_LOG "LOOC log"
 
-#define ADMIN_MARKREAD(client) "(<a href='?_src_=holder;markedread=\ref[client]'>MARK READ</a>)"//marks an adminhelp as read and under investigation
-#define ADMIN_IC(client) "(<a href='?_src_=holder;icissue=\ref[client]'>IC</a>)"//marks and adminhelp as an IC issue
-#define ADMIN_REJECT(client) "(<a href='?_src_=holder;rejectadminhelp=\ref[client]'>REJT</a>)"//Rejects an adminhelp for being unclear or otherwise unhelpful. resets their adminhelp timer
\ No newline at end of file
+GLOBAL_VAR_INIT(CITADEL_MENTOR_OOC_COLOUR, "#FF3E96")
+#define R_MENTOR 8192
+#define R_MAXPERMISSION 8192
```
Comment (silicons) [inline code/__DEFINES/citadel_defines.dm:73]: shouldn't this be all permissions combined?

**Comment by deathride58** [comment]
Finish this up, dork

**Pull request closed without merging**